src/main/java/org/jenkinsci/plugins/docker/workflow/RegistryEndpointStep.java

```diff
@@ -105,6 +111,28 @@ public DescriptorImpl() {
             return true;
         }
 
+        @Override public UninstantiatedDescribable uninstantiate(Step step) throws UnsupportedOperationException {
+            RegistryEndpointStep s = (RegistryEndpointStep) step;
+            Map<String, Object> args = new TreeMap<>();
+            args.put("url", s.registry.getUrl());
+            args.put("credentialsId", s.registry.getCredentialsId());
+            args.put("toolName", s.toolName);
+            args.values().removeAll(Collections.singleton(null));
```

Comment:
❗️ Will this throw a ConcurrentModificationException if you remove a value and then iterate another item?